docs/attrStart.md

```diff
@@ -4,4 +4,8 @@
 
 Called when an attribute is found on an element.
 
+Handles encoding of certain characthers:
+1. Spaces (` `) - Encoded to `\s`.
+1. Forward Slashes (`/`) - Encoded to `\fs`.
```

Comment:
I don't think it should encode to a two space character.  Does `\/` work? Or even `\F` or `\f`